qemu/tests/cfg/virtio_fs_multi_users_access.cfg

```diff
@@ -0,0 +1,46 @@
+- virtio_fs_multi_users_access:
+    type = virtio_fs_multi_users_access
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir = virtio_fs_test/
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    driver_name = viofs
+    fs_target = 'myfs'
+    fs_dest = "/mnt/${fs_target}"
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
```

Comment:
@zhenyzha Could you help check if multi-arch also support this scenario? cc @nickzhq

Reply:
Many thanks for the updated info :)
Yes, this userspace scenario does not differ between platforms.
The code is also clear to me,

` (1/1) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.virtio_fs_multi_users_access.localgroup_users.arm64-pci: PASS (142.55 s)`

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>